Packs/BmcITSM/README.md

```diff
@@ -0,0 +1,9 @@
+Cortex XSOAR interfaces with BMC Helix ITSM to help streamline security-related service management and IT operations.
+
+# What does this pack do?
+ - View, create, update, or delete a BMC Helix ITSM ticket directly from the Cortex XSOAR.
+ - View, create, update, and delete records from other BMC Helix ITSM tables: Users, Companies, and Ticket templates. 
```

Comment:
- Views, creates, updates, and deletes records from BMC Helix ITSM tables: Users, Companies, and Ticket templates.